### PR TITLE
Testing Calculation logic and Components

### PR DESCRIPTION
**Pull request by edenlisk**
In this PR, We did the following:
1. Test operate.js
2. Test calculate.js
3. Test all pages components

### CODE REVIEW

File: src/__test__/components.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    expect(author).toHaveTextContent('Welcome to our page');
+  });
+});
+
+describe('Test different components', () => {
+  test('It should render Quote component', () => {
+    render(<Quote />);
+    expect(screen.getByText(/William Paul Thurston/)).toBeInTheDocument();
+  });
+});
```
Comment (danmainah) [inline src/__test__/components.test.js]: - [x] @edenlisk and @collins-kiprotich great work on your components test, however, you need to add tests for all components , didn't see tests for the app navbar and calculator components, this is a project requirement
image

File: src/__test__/logic.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    const solution = calculate(container, 'AC');
+    expect(solution).toEqual({ total: null, next: null, operation: null });
+  });
+
+  test('Test inverse sign (+/-)', () => {
+    const container = { total: 70, next: null, operation: null };
+    const solution = calculate(container, '+/-');
+    expect(solution).toEqual({ total: '-70', next: null, operation: null });
+  });
+});
```
Comment (danmainah) [inline src/__test__/logic.test.js]: - [x] @team I would recommend you separate your test for each file eg tests for operate.js and calculate.js should be separate, kindly ensure you cover all functionalities & scenarios in the operate and calculate js files, they are separated using the if statement

Comment (danmainah) [reply src/__test__/logic.test.js]: - [x] @team I observed that you have disabled the linter check in your App component, its not good practise to disable linters I would suggest you try and fix it 
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @edenlisk and @collins-kiprotich , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by pinkmoon25 — APPROVED** [review]
Hi @edenlisk and @collins-kiprotich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter check :heavy_check_mark: 
GitHub flow followed :heavy_check_mark: 
Readme :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**